test/cli/test_cli_flow.sh

```diff
@@ -64,6 +64,7 @@ function clean {
 }
 
 function main {
+    crd_cycle
```

Comment:
Why crd cycle is here before the install?
How should we check the crd cycle if there is no NooBaa?

Reply:
I consulted with Ohad and we thought its the best way, after noobaa is created it tries to delete crd while its not empty

Reply:
we can run this cycle prior to the install but also need to do it after the install and exclude the parts that we don't want to run.